tests/providers/amazon/aws/transfers/test_sql_to_s3.py

```diff
@@ -145,16 +145,30 @@ def test_execute_json(self, mock_s3_hook, temp_mock):
                 replace=True,
             )
 
-    def test_fix_dtypes(self):
+    def test_fix_dtypes_csv(self):
         op = SqlToS3Operator(
             query="query",
             s3_bucket="s3_bucket",
             s3_key="s3_key",
             task_id="task_id",
             sql_conn_id="mysql_conn_id",
         )
-        dirty_df = pd.DataFrame({"strings": ["a", "b", "c"], "ints": [1, 2, None]})
-        op._fix_dtypes(df=dirty_df)
+        dirty_df = pd.DataFrame({"strings": ["a", "b", None], "ints": [1, 2, None]})
+        op._fix_dtypes(df=dirty_df, file_format="csv")
+        assert dirty_df["strings"].values[2] is None
+        assert dirty_df["ints"].dtype.kind == "i"
+
+    def test_fix_dtypes_parquet(self):
```

Comment:
You can parameterize it rather than creating a new test

Reply:
example: https://github.com/apache/airflow/blob/9e06c99f6102d0227c6e7b20b258d628c2bc6d5c/tests/sensors/test_weekday_sensor.py#L56-L78

Reply:
good callout, fixed that up if you want to take another look!